packages/@okta/vuepress-site/docs/guides/anything-as-a-source/index.md

```diff
@@ -0,0 +1,374 @@
+> [Van question]: If this known issue is still valid, then can we say that there can only be one Import Session in the `CREATED` status for an identity source? And remove the 5 minute note in the above description?
+
+### Bulk user imports
+
+You can upload up to 200 KB of data in a single bulk import request for a particular Import Session. This equates to 200 user profiles. To add more user profiles, you can make multiple bulk import requests for a particular session. The maximum number of bulk import requests for a session is 50. If you have exhausted the maximum number of bulk import requests and you still need to upload more user profiles, then create another Import Session object for the additional user profiles. Keep in mind that you can only import user data for Import Sessions in the `CREATED` status and that you can’t process user imports in parallel for a specific identity source.
+
+### Bulk user profile data
+
+The bulk import request contains an array of [User Profile Data](/docs/reference/api/xaas/#profile-object) objects that contain the following:
+
+* `externalId`: Contains the unique identifier from the HR source and is assumed to be immutable (never updated for a specific user). This value is used as a key to determine if a new user needs to be created or an existing user needs to be updated.
+
+* `profile`:  Contains the set of attributes from the HR source to synchronize with Okta user profiles. User profiles are mapped according to the attribute mappings you specified in your Custom Identity Source configuration.  See Declaration of a Custom Identity Source Schema in  [Using anything as a source](https://help.okta.com/okta_help.htm?type=wf).
+
+> **Note:** Only user profile data can be imported to an Import Session object with the `entityName` = `USERS` property. Group data import isn’t supported.
+
+### Bulk user profile data type
+
+There are two types of bulk import requests:
+
+* `UPSERT`: Insert or update the profile users in this bulk import request
+* `DELETE`: Deactivate the user profiles in this bulk import request
+
+## XaaS API flow
+
+Before you start to build your integration component, you need to set up a few configuration variables:
+
+* Your Okta org domain URL (`${yourOktaDomain}`): This is for XaaS API requests.
+* Your Custom Identity Source ID (`${identitySourceId}`): This is the unique identifier you obtain from [configuring a Custom Identity Source integration] in your Okta org.
+* An API Token (`${apiKey}`}: You need to [obtain an API token from your Okta org] to make API calls to Okta. Use this API token in the SSWS Authorization header.
+
+Code your XaaS data synchronization component with the following generalized API flow:
+
+1. Create an Import Session.
+2. Load user data profile into the Import Session (if required, load multiple batches of user data).
+3. Trigger the data import process.
+4. Monitor the Import Session until the data processing completes.
+5. Close the Import Session.
```

Comment:
No.  This is wrong.  Once the data processing is complete, the session will automatically go into the COMPLETED status.  Customers doesn't need to close the session exclusively.

Once the session is triggered (to start the data processing), it cannot be closed or cancelled.

Before the session is triggered, client can cancel the session using the API 

DELETE /api/v1/identity-sources/{{identitySourceId}}/sessions/{{sessionId}}